lib/File/Find/Rule.pm

```diff
@@ -485,6 +485,18 @@ sub relative () {
     $self;
 }
 
+=item C<normalize>
+
+Normalize paths found using C<File::Spec->canonpath>
```

Comment:
I read the File::Spec docs and the behaviour you seem to want, ensuring that the most native looking path separator is used, is not advertised as a feature to `canonpath`, so I'm going to say that for the change you're trying to make handwaving and saying 'it's well understood' doesn't cut it.